app/views/hyrax/users/index.html.erb

```diff
@@ -1,21 +1,50 @@
-<div id="search">
-  <%= render 'search_form' %>
-  <h1><%= application_name %> Users</h1>
-</div>
-<table class="table table-striped">
-    <thead>
-        <tr>
-            <th>Avatar</th>
-            <th class="sorts"><i id="name" class="<%= params[:sort].blank? ? 'caret up' : params[:sort] == "name desc" ? 'caret' : params[:sort] == "name" ? 'caret up' : '' %>"></i> User Name</th>
-            <th class="sorts"><i id="login" class="<%= params[:sort] == "login desc" ? 'caret' : params[:sort] == "login" ? 'caret up' : '' %>"></i> User Id</th>
-            <th class="sorts"><i id="department" class="<%= params[:sort] == "department desc" ? 'caret' : params[:sort] == "department" ? 'caret up' : ''%>"></i> Department</th>
-            <th>Works Created</th>
-        </tr>
-    </thead>
-    <tbody>
-    <%= render partial: 'user_row', collection: @users, as: :user %>
-    </tbody>
-</table>
-<div class="pager">
-  <%= paginate @users, theme: 'blacklight', route_set: hyrax %>
+<% content_for :page_header do %>
+  <h1><span class="fa fa-user"></span> <%= t('hyrax.users.index.title') %></h1>
+<% end %>
+
+<div class="panel panel-default users-listing">
+  <div class="panel-heading">
+    <% count_users = @presenter.user_count %>
+    <% admin_count = @presenter.repository_admin_count %>
+    <% if count_users == 1 %>
```

Comment:
Yes, unlikely to happen in reality, though it could happen depending on how tests are set up. I believe I mimicked the logic from the way it was showing elsewhere, but I'm find with putting it into the presenter. That definitely would simplify the view.